homeassistant/util/color.py

```diff
@@ -732,3 +734,38 @@ def check_valid_gamut(Gamut: GamutType) -> bool:
     )
 
     return not_on_line and red_valid and green_valid and blue_valid
+
+
+def brightness_to_value(low_high_range: tuple[float, float], brightness: int) -> float:
```

Comment:
Which other integration would benefit form this util functions?

Reply:
Any hub integration that implements lights that support brightness. To start the scaling code is shared.

Reply:
Need to do some research, the assumption is more based on @emontnemery 's chat from Discord.

Reply:
If I just search on `kwargs[ATTR_BRIGHTNESS]` I find a lot of integrations that could benefit from the util functions.